master/internal/core_experiment.go

```diff
@@ -563,6 +563,12 @@ func (m *Master) parseCreateExperiment(params *CreateExperimentParams, user *mod
 		dbExp.Username = user.Username
 	}
 
+	taskSpec.Project = *config.RawProject
```

Comment:
call `config.Project()` instead. same for other fields

Reply:
Fixed. Thanks!